compiler/rustc_type_ir/src/visit.rs

```diff
@@ -230,7 +230,7 @@ pub trait TypeVisitableExt<I: Interner>: TypeVisitable<I> {
     }
 
     fn has_aliases(&self) -> bool {
```

Comment:
```suggestion
     fn has_alias(&self) -> bool {
```

too (my bad)

Reply:
interstingly, we do use the plural for the functions, e.g. we have `has_inherent_projections` and `has_opaque_types` even though the type flag is `HAS_TY_OPAQUE`

Reply:
let's leave this one